1.3 Metrics - Adding Callout For Ephemeral Models (#2108)
fishtown-analytics/docs.getdbt.com: website/docs/docs/building-a-dbt-project/metrics.md

</File>   

:::caution

> - You can not defined metrics on [ephemeral models](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/materializations#ephemeral). To define a metric, the materialization must have a represention in the data warehouse.

Reviewer: sorry @callum-mcdata , fixed the two typos here!
```suggestion
- You cannot define metrics on [ephemeral models](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/materializations#ephemeral). To define a metric, the materialization must have a representation in the data warehouse.
```

> - Metrics cannot be built on [ephemeral models](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/materializations#ephemeral). The materialization must have a represention in the data warehouse in order to build a metric.

Reviewer: switch to active voice and fix representation typo

```suggestion
- You cannot build metrics on [ephemeral models](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/materializations#ephemeral). To build a metric, the materialization must have a representation in the data warehouse.
```

Author: Actually scratch that, I'm going to change built to define.
